README.md

```diff
@@ -16,32 +16,32 @@
 
 ## Executive summary
 
-- Immuni is a technological solution that centres on **an iOS and Android smartphone app.** It helps us fight the COVID-19 pandemic by notifying users at risk of carrying the virus as early as possible—**even when they are asymptomatic.** These users can then isolate themselves to avoid infecting others, and seek medical advice.
-- Immuni’s design and development are founded on five main principles: **utility, accuracy, scalability, transparency,** and **privacy.**
+- Immuni is a technological solution that centres on **an iOS and Android smartphone app.** It helps us to fight the COVID-19 pandemic by notifying users at risk of carrying the virus as early as possible—**even when they are asymptomatic.** These users should then isolate themselves to avoid infecting others, and seek medical advice.
+- Immuni’s design and development are based on five main principles: **utility, accuracy, scalability, transparency,** and **privacy.**
 - It features a **contact tracing** system based on **Bluetooth Low Energy:**
-  - When two users come sufficiently close to each other for long enough, their devices record each other’s _rolling proximity identifier_ in their local memory. These identifiers are generated from _temporary exposure keys_ and change multiple times an hour. These keys are **generated randomly** and change once per day.
+  - When two users come sufficiently close to each other for long enough, their devices record each other’s _rolling proximity identifier_ in their local memory. These identifiers are generated from _temporary exposure keys_ and change multiple times each hour. These keys are **generated randomly** and change once per day.
   - When a user tests positive for SARS-CoV-2, the virus causing COVID-19, they have the option to upload to a server their recent temporary exposure keys. This operation can only happen with the validation of a **healthcare operator.**
   - The app periodically downloads the new temporary exposure keys and uses them to derive the infected users’ rolling proximity identifiers for the recent past. It then matches them against those stored in the device’s memory and **notifies the user** if a risky contact has occurred.
-  - The system uses **no geolocation data** whatsoever, including GPS data. So, the app cannot tell where the contact with a potentially contagious user took place, nor the identities of those involved.
+  - The system **does not use any geolocation data** whatsoever, including GPS data. So, the app cannot tell where the contact with a potentially contagious user took place, nor the identities of those involved.
```

Comment:
Our copywriters believe the existing language works better in British English. It also has the advantage of minimizing the amount of bold text.

Reply:
Since most of your audience is Italian, I thought that *use not ..* is way stronger than *uses no ...*. It'd be easier to get people confused and maybe say the wrong things about the app :)

Reply:
There's certainly been some confusion. :) Thanks for being so thoughtful.n the